webapp/src/pages/EditChatroomMemberPage/components/ToggleSwitchSet.tsx

```diff
@@ -97,29 +97,19 @@ export default function ToggleSwitchSet({
           ...updateChatroomMemberDto,
           [key]: newValue,
         });
-        const { username } = canEditParams.destUser;
-        try {
-          notify(
-            newValue
-              ? `Added ${key} role to ${username}`
-              : `Removed ${key} role from ${username}`,
-          );
-        } catch {
-          notify(`${username} chatroom member successfully updated`);
-        }
+        notify(
```

Comment:
we needed the try catch before because we to get the key we had a helper function that could throw. Not anymore: that code path is obsolete and doesn't exist anymore.